Makefile

```diff
@@ -71,15 +78,20 @@ check: python-env
 # Corrects spelling errors
 .PHONY: misspell
 misspell:
-	go get github.com/client9/misspell
-	${FIND} -name '*'  -exec misspell -w {} \;
+	go get $(MISSPELL_REPO)
+	$(FIND) -name '*'  -exec $(MISSPELL) -w {} \;
 
 .PHONY: fmt
 fmt: python-env
 	$(foreach var,$(PROJECTS),$(MAKE) -C $(var) fmt || exit 1;)
 	# Cleans also python files which are not part of the beats
 	find . -type f -name *.py -not -path "*/vendor/*" -not -path "*/build/*" -not -path "*/.git/*" -exec autopep8 --in-place --max-line-length 120  {} \;
 
+.PHONY: lint
```

Comment:
I'm suggesting moving the implementation of the `lint` target to `libbeat/scripts/Makefile`. This provides two things:

- Developers of community beats can run `make lint` to see new linter warnings that they introduced while making changes.
- Developers of elastic/beats can call `make lint` inside of a beat sub-dir and the operation is limited to changes to that specific beat.

I would still want to have the ability to run `make lint` at the top level in `elastic/beats` to check the whole repo. This is consistent with other commands like `make fmt` or `make check`.